tools/create-genesis/src/miner.rs

```diff
@@ -0,0 +1,51 @@
+use crate::common::{get_temp_state, secret_2_address, string_2_bytes, string_2_u256};
+use crate::genesis::Account;
+use core_executor::engines::NullEngine;
+use core_executor::executive::contract_address;
+use core_executor::libexecutor::sys_config::BlockSysConfig;
+use core_executor::types::transaction::Transaction;
+use evm::cita_types::U256;
```

Comment:
Why use `emv::cita_types`? directly from cita-common or use ethereum-types?
I feel a little weired?

Reply:
I can not  use ethereum-types here for the reason of different version. 
Using `evm::cita_types` or cita-common, it's all works well. Using `cita-common cita_types` needs import `cita-types` dependently in Cargo.toml, whille use `evm::cita_types` need no more extra import since we have imported evm.

Reply:
OK. still the problem of chaos of cita-types and ethereum-types

Reply:
Yeal, i am trying to replace all ethereum-types in cita-vm with numexe, all kind of problems.

Some breaking changes in ethereum-types crate different version.

Reply:
got to pay the debts